src/fmu/dataio/export/rms/_utils.py

```diff
@@ -92,6 +92,19 @@ def validate_global_config(config: dict[str, Any]) -> GlobalConfiguration:
         ) from err
 
 
+def get_open_polygons_id(pol: xtgeo.Polygons) -> list[int]:
```

Comment:
Perhaps XTGeo should have something like this?

Reply:
Yes, that would be nice to get this as a method on the polygons object from XTGeo 👍. I see in XTGeo there already are some code for it, since some methods require closed polygons. :) just a matter of exposing it to the users.